smoke-tests/apps/HttpPreaggregatedMetrics/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/HttpPreaggregatedMetricsTest.java

```diff
@@ -162,9 +162,15 @@ private static void validateMetricData(
     DataPoint dataPoint = dataPoints.get(0);
     assertThat(dataPoint.getCount()).isEqualTo(1);
     double greaterThan = type.equals("server") ? 200.0f : 0.0f;
-    assertThat(dataPoint.getValue()).isGreaterThan(greaterThan).isLessThan(60 * 1000.0);
-    assertThat(dataPoint.getMin() * 1000).isGreaterThan(greaterThan).isLessThan(60 * 1000.0);
-    assertThat(dataPoint.getMax() * 1000).isGreaterThan(greaterThan).isLessThan(60 * 1000.0);
+    assertThat(dataPoint.getValue())
+        .isGreaterThan(greaterThan)
+        .isLessThan(70 * 1000.0); // wildfly takes longer
```

Comment:
(probably can go back to 60?)
```suggestion
        .isLessThan(60 * 1000.0);
```